src/main/java/io/druid/data/input/impl/JSONParseSpec.java

```diff
@@ -30,15 +32,23 @@
 public class JSONParseSpec extends ParseSpec
 {
   private final ObjectMapper objectMapper;
+  private final JSONPathSpec flattenSpec;
 
   @JsonCreator
   public JSONParseSpec(
       @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
-      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec
+      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec,
+      @JsonProperty("flattenSpec") JSONPathSpec flattenSpec
```

Comment:
any libraries which expect to find a (S, S)V will fail to link here. this would require a major version update